packages/code-studio/src/settings/FormattingSectionContent.jsx

```diff
@@ -191,6 +191,9 @@ export class FormattingSectionContent extends PureComponent {
     this.handleResetDecimalFormat = this.handleResetDecimalFormat.bind(this);
     this.handleResetIntegerFormat = this.handleResetIntegerFormat.bind(this);
     this.handleResetTimeZone = this.handleResetTimeZone.bind(this);
+    this.handletruncateNumbersWithPoundChange = this.handletruncateNumbersWithPoundChange.bind(
```

Comment:
Change to `handleTruncateNumbersWithPoundChange` (casing on `t`)

Reply:
Good catch